### PR TITLE
Add parts-per-{cent,million,billion,trillion,quadrillion}.

### PR DESCRIPTION
**Pull request by jimmo**

### CODE REVIEW

File: src/Data/Units/PartsPer.purs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+-- | A collection of parts-per-x units.
+module Data.Units.PartsPer where
```
Comment (sharkdp) [inline src/Data/Units/PartsPer.purs]: Minor suggestion: maybe call this `PartsPerX`?

File: src/Data/Units/PartsPer.purs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import Prelude
+
+import Data.Units (DerivedUnit, makeNonStandard, unity)
+
+-- | 'Dimensionless' ratio *1 pct = 1e-2*.
+pct ∷ DerivedUnit
```
Comment (sharkdp) [inline src/Data/Units/PartsPer.purs]: I think we should use long names for the units here, i.e. `percent` and `partsPerMillion`. For consistency with other modules in this library.

**Comment by sharkdp** [comment]
Thank you very much, that looks like a useful addition!

File: src/Data/Units/PartsPer.purs (hunk 2 above)
Comment (jimmo) [reply src/Data/Units/PartsPer.purs]: Sure thing. Updated.

File: src/Data/Units/PartsPer.purs (hunk 1 above)
Comment (jimmo) [reply src/Data/Units/PartsPer.purs]: Done.

**Comment by jimmo** [comment]
> Thank you very much, that looks like a useful addition!

No worries! Was excited to find this project, have found it very useful already -- thank you :)

**Comment by sharkdp** [comment]
Thank you for the update

**Pull request merged**